<launch>

  <node pkg="rosbag" type="play" name="player" args="--clock /home/adrl/bagfiles/slow_motion.bag">
    <remap from="cam0/image_raw" to="stereo/left/image_raw"/>
    <remap from="cam0/camera_info" to="stereo/left/camera_info"/>
    <remap from="cam1/image_raw" to="stereo/right/image_raw"/>
    <remap from="cam1/camera_info" to="stereo/right/camera_info"/>
  </node>

  <group ns="stereo"> 
 
  <node pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_proc"/>

  </group>

  <node pkg="image_view" type="image_view" name="single_image">
    <remap from="image" to="/stereo/right/image_mono"/>
  </node>

<!--node pkg="image_view" type="stereo_view" name="stereo_image">
    <remap from="stereo" to="/stereo"/>
    <remap from="disparity" to="/stereo/disparity"/>
  </node-->

</launch>

